### DISCUSSION TITLE
（Httpapi）返回400 无效参数

### DISCUSSION THREAD

**Discussion by Hikari16665** [DreamVoid/MiraiMC#444] [问答 / Q&A]
我使用miraimc的httpapi连接到我部署的mcl服务，正常连接。
在使用sendgroupmessage发送消息的时候，显示400 无效参数。
由于我没能力查代码之类的，所以就来问问。这个httpapi似乎请求方式有点改变，要提供一个叫sessionkey的东西，这个东西还要用authkey申请，问问miraimc有没有适配这个东西。

**Comment by DreamVoid** [comment]
什么日志图片都没有，你让我分析什么，分析你今天吃了什么菜吗

**Reply by Hikari16665** [reply]
部分日志：
https://mclo.gs/UD14jKN

**Reply by Hikari16665** [reply]
刚找到了那个httpapi的代码，大概能看懂一点，感觉也没啥问题，总之 求大佬解答，谢谢！

**Reply by Hikari16665** [reply]
如果需要的话我也可以把我mcl的api地址发给您
